handbook/engineering/README.md

```diff
@@ -36,6 +36,15 @@ To [create an engineering-initiated user story](https://fleetdm.com/handbook/eng
 
 > We prefer the term engineering-initiated stories over technical debt because the user story format helps keep us focused on our users and contributors.
 
+### Manage release branches
+Every three weeks, release a minor version of Fleet from the `main` branch. 
+
+Every week between minor releases, release a patch release with fixes for released bugs. 
+
+When the next release is a patch, create a patch branch off the latest tagged release of Fleet. Merge released bug fixes directly into the target patch branch to avoid merge conflicts later in the release cycle. After merging into the patch branch, submit a PR to `main` containing the same fix and resolve any merge conflicts.
```

Comment:
Missing - who is the DRI for creating the patch branch?
The previous release owner?

Reply:
Q2 - should we use the same patch branch for both patch releases of the sprint?
or close the first one immediately after releasing Patch-I ?

Reply:
Good point re: DRI, updated. 

Yes, we will create a new patch branch for every release to keep things organized.